easybuild/easyconfigs/c/CLHEP/CLHEP-2.4.1.3-intel-2019b.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'CMakeMake'
+
+name = 'CLHEP'
+version = '2.4.1.3'
+
+homepage = 'http://proj-clhep.web.cern.ch/proj-clhep/'
+
+description = """
+ The CLHEP project is intended to be a set of HEP-specific foundation and
+ utility classes such as random generators, physics vectors, geometry and linear algebra.
+ CLHEP is structured in a set of packages independent of any external package.
+"""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://proj-clhep.web.cern.ch/proj-clhep/dist1']
+sources = [SOURCELOWER_TGZ]
+checksums = ['27c257934929f4cb1643aa60aeaad6519025d8f0a1c199bc3137ad7368245913']
+
+builddependencies = [('CMake', '3.15.3')]
+
+separate_build_dir = True
+
+configopts = '-DCMAKE_BUILD_TYPE=Release'
+
```

Comment:
@JackPerdue Test suite fails because this `CLHEP` easyconfig is using a generic easyblock but doesn't specify a custom `sanity_check_paths`, please add one?
That's a new requirement since a while...

Reply:
Ack.  Will do.